src/defaults.js

```diff
@@ -9,7 +9,7 @@ export function getDefaults() {
     headerPrefix: '',
     highlight: null,
     hooks: null,
-    langPrefix: 'language-',
+    langPrefix: null,
```

Comment:
Technically this is semver major because its changing the public API

```js
import marked from 'marked';
const result = marked.getDefaults();
```

Reply:
Have we considered using (and is it possible to use) empty types instead of `null`?

Coming at it from someone who doesn't know the architecture and intent, I'd need to search and find what's expected from the argument list, yeah?

Reply:
That is true. And any extensions that rely on `options.langPrefix` could break.

Reply:
I changed this PR to only check if langPrefix has changed from it's default value before warning about it.